xdsl/dialects/x86/ops.py

```diff
@@ -688,11 +741,11 @@ class RM_XorOp(R_RM_Operation[GeneralRegisterType, GeneralRegisterType]):
 
 
 @irdl_op_definition
-class RM_MovOp(R_RM_Operation[GeneralRegisterType, GeneralRegisterType]):
+class RM_MovOp(R_M_Operation[GeneralRegisterType, GeneralRegisterType]):
     """
-    Copies the value from the memory location pointed to by r2 into r1.
+    Copies the value from the memory location pointed to by source register into destination register.
     ```C
-    x[r1] = [x[r2]]
+    x[output] = [x[operand]]
```

Comment:
Can you please adjust these based on r1 and r2 as per the rest of the file; I don't understand what `output` or `operand` is as they appear nowhere else in the API?